plot-api/src/jsMain/kotlin/jetbrains/letsPlot/Env.kt

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright (c) 2022. JetBrains s.r.o.
+ * Use of this source code is governed by the MIT license that can be found in the LICENSE file.
+ */
+
+package jetbrains.letsPlot
+
+internal actual object Env {
+    actual fun get(name: String): String? {
+        TODO("Not yet implemented")
```

Comment:
Why not return null?

Reply:
Was thinking about using [localStorage](https://kotlinlang.org/api/latest/jvm/stdlib/kotlinx.browser/local-storage.html) as Env in JS. For now will return `null`.